R/pkg/R/DataFrame.R

```diff
@@ -692,8 +696,8 @@ setMethod("sample",
 setMethod("sample_frac",
           signature(x = "DataFrame", withReplacement = "logical",
                     fraction = "numeric"),
-          function(x, withReplacement, fraction) {
-            sample(x, withReplacement, fraction)
+          function(x, withReplacement, fraction, seed) {
+              sample(x, withReplacement, fraction, as.integer(seed))
```

Comment:
Then, you need to change the test case. If we do not use `as.integer(seed)`, we need to change the input type. For example, 
```
sampled2 <- sample(df, FALSE, 0.1, 0) 
```
needs to be changed to
```
sampled2 <- sample(df, FALSE, 0.1, 0L) 
```